etc/packaging/appimages/rplidar-module-aarch64.yml

```diff
@@ -33,6 +33,7 @@ AppDir:
     - libstdc++6:arm64
     - libc6:arm64
     - libgcc-s1:arm64
+    - libjpeg-dev:arm64
```

Comment:
please also update the x86_64 app image yaml file in this same directory & confirm you are able to reproduce the error on x86_64 canon & that the yaml file change fixes.

this is how you tell canon to give you an x86_64 container to test with:
```
canon -arch=arm64
```